dtests/build.gradle

```diff
@@ -89,16 +96,17 @@ archivesBaseName = 'gemfirexd-scala-tests'
 
 testClasses.doLast {
     copyTestsCommonResources(buildDir)
-}
-
-copy {
-    def osDir = osName.getFamilyName().replace(" ", "").toLowerCase()
-    from (rootDir.getAbsolutePath() + '/dtests/src/test/java') {
-        include '**/*.bt'
-        include '**/*.conf'
-        include '**/*.inc'
-        include '**/*.sql'
+    copy {
+        def osDir = osName.getFamilyName().replace(" ", "").toLowerCase()
```

Comment:
Yes..I have seen these changes in master
I have not merged master yet. I am planning to merge master once I am done testing these changes on cluster